File: 2186-minimum-number-of-steps-to-make-two-strings-anagram-ii/2186-minimum-number-of-steps-to-make-two-strings-anagram-ii.java
class Solution {
    public int minSteps(String s, String t) {
        HashMap<Character, Integer> mp = new HashMap<>();
        for(char c : s.toCharArray()) {
            mp.put(c, mp.getOrDefault(c, 0) + 1);
        }
        for(char c : t.toCharArray()) {
            mp.put(c, mp.getOrDefault(c, 0) - 1);
        }
        int count = 0;
        for(int value : mp.values()) {
            count += Math.abs(value);
        }
        return count;
    }
}

File: 2186-minimum-number-of-steps-to-make-two-strings-anagram-ii/2186-minimum-number-of-steps-to-make-two-strings-anagram-ii.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        map<char,int> mp;
        for(char c : s) {
            mp[c]++;
        }
        for(char c : t) {
            mp[c]--;
        }
        int count = 0;
        for(auto it : mp) {
            count += abs(it.second);
        }
        return count;
    }
};